src/Symfony/Component/Console/Application.php

```diff
@@ -1002,6 +1002,8 @@ private function findAlternatives($name, $collection)
      *
      * @param string $commandName     The Command name
      * @param bool   $isSingleCommand Set to true if there is only one command in this application
+     *
+     * @return Application The current application
```

Comment:
can be `self` without extra comment since it's not any implementation but the "current"